packages/@aws-cdk/aws-codepipeline-actions/test/cloudformation/integ.stacksets.ts

```diff
@@ -0,0 +1,60 @@
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { Asset } from '@aws-cdk/aws-s3-assets';
+import { App, Stack, StackProps } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as cpactions from '../../lib';
+
+export class StackSetPipelineStack extends Stack {
+  constructor(scope: Construct, id: string, props: StackProps) {
+    super(scope, id, props);
+
+    const pipeline = new codepipeline.Pipeline(this, 'Pipeline');
+
+    const asset = new Asset(this, 'Asset', {
+      path: `${__dirname}/test-artifact`,
+    });
+
+    const sourceOutput = new codepipeline.Artifact('SourceArtifact');
+
+    pipeline.addStage({
+      stageName: 'Source',
+      actions: [
+        new cpactions.S3SourceAction({
+          actionName: 'Source',
+          output: sourceOutput,
+          bucket: asset.bucket,
+          bucketKey: asset.s3ObjectKey,
+        }),
+      ],
+    });
+
+    const accounts = process.env.STACKSET_ACCOUNTS?.split(',') ?? ['1111', '2222'];
```

Comment:
What's happening here?

Reply:
To actually correctly run it across multiple accounts I need to be able to configure actual accounts that I have actual access to. I'm not putting them in the source, so they are environment variables.

Reply:
Sorry, I still don't understand 😛.

The expected template contains this:

```json
                  "DeploymentTargets": "1111,2222",
```

And the expected template what was deployed, right? If you just pass things here in an env variable, these will not match what's in the expected file, no?

I feel like I'm missing something fundamental here, which probably means this deserves at least a comment in the code 🙂.